### ISSUE TITLE
Pain on wrist deviation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10742]
**Preferred term label:**
Pain worsens with wrist ulnar deviation
Pain worsens with wrist radial deviation

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ulnar:
Triangular fibrocartilage complex (TFCC) tear
Ulnocarpal abutment syndrome

Radial:
Triangular fibrocartilage complex (TFCC) tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001152
- label: Pain exacerbated by wrist deviation
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: 
- def: Pain worsens with wrist ulnar or radial deviation.
- comment: Ulnar: Triangular fibrocartilage complex tear; Ulnocarpal abutment syndrome Radial: Triangular fibrocartilage complex tear.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10742
- synonyms:

**Comment by MickeySegal** [comment]
This was meant as a request for 2 child terms of the term that you created.  The direction of the wrist deviation causing the pain is important diagnostically.

- Pain exacerbated by wrist ulnar deviation
- Pain exacerbated by wrist radial deviation

**Comment by pnrobinson** [comment]
reopening

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001419
- label: Pain exacerbated by wrist ulnar deviation
- parents: [Pain exacerbated by wrist deviation (HP:6001152)](https://hpo.jax.org/app/browse/term/HP:6001152)
- pmid: 
- def: Pain worsens with wrist ulnar deviation (i.e., the pain is exacerbated if the hand is moved towards the side with the fifth finger).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10742
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001420
- label: Pain exacerbated by wrist radial deviation
- parents: [Pain exacerbated by wrist deviation (HP:6001152)](https://hpo.jax.org/app/browse/term/HP:6001152)
- pmid: 
- def: Pain worsens with wrist radial deviation (i.e., the pain is exacerbated if the hand is moved towards the side with the thumb).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10742
- synonyms:
